src/components/sync-configurator/OrganisationUnitSync.jsx

```diff
@@ -3,16 +3,58 @@ import BaseSyncConfigurator from "./BaseSyncConfigurator";
 import PropTypes from "prop-types";
 import i18n from "@dhis2/d2-i18n";
 import { OrganisationUnitModel } from "../../models/d2Model";
+import Dropdown from "../shared/Dropdown";
 
 export default class OrganisationUnitSync extends React.Component {
     static propTypes = {
         d2: PropTypes.object.isRequired,
     };
 
+    state = {
+        dropdown: {
```

Comment:
Name of property should reflect the what (orgUnitGroup?) not the how (that is a dropdown element)